src/utils/render-utils.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import ExternalLink from '../shared/components/links/ExternalLink';
+
+const URL_REGEXP =
+  /https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_+.~#?&/=]*)/;
+export const GROUP_MATCH_REGEXP = new RegExp(`^(.*\\s)?(${URL_REGEXP.source})(\\s.*)?$`, 'i');
+
+export const handleURLs = (value: string): React.ReactNode => {
+  if (typeof value !== 'string') return value;
+
+  const matches = value.match(GROUP_MATCH_REGEXP);
```

Comment:
This regex covers only `https` urls, shall we include regex to match `quay.io` as well, so that we can link it to the quay registry?

<img width="813" alt="image" src="https://user-images.githubusercontent.com/9964343/215397694-b515b0a5-f2b6-4440-8cd3-de79d6c563f8.png">

Reply:
@karthikjeeyar I think this may be worth looking into via a separate issue because changing this into a URL doesn't actually bring us to a useful page.

Reply:
Ah I see, tag page does not exist.